### DISCUSSION TITLE
catalog-items-api-model

### DISCUSSION THREAD

**Discussion by ayotade** [abuzuhri/Amazon-SP-API-CSharp#519] [Q&A] [answered]
Hell,

I am unable to find  attributes below in the catalog-items-api-model and my application is requesting them. Please help

unit_count
power_plug_type
eu_energy_label_efficiency_class
efficiency
accepted_voltage_frequency
item_length_width_height

Thank you in anticipation of helping me @abuzuhri  @RenzoF  @ProNotion @Paul-J-Barrett @shashi-attri

**Comment by abuzuhri** [comment]
Please share code attached here and i will try help you this weekend

**Comment by ayotade** [comment]
PetplanetAmazonApp.zip
<https://drive.google.com/file/d/16rG2zVW_lRplYpHwBlzEwqla0--bqifL/view?usp=drive_web>

Please find attached zip file.

thank you very much.

**Comment by abuzuhri** [comment]
still no permission to download the file

**Comment by ayotade** [comment]
Please try now.

**Comment by ayotade** [comment]
PetplanetAmazonApp.zip
<https://drive.google.com/file/d/1ZsC6ZjqyMnFBLWQBukcDnrJ0VqjgoD_A/view?usp=drive_web>

**Comment by ayotade** [comment]
https://drive.google.com/file/d/1ZsC6ZjqyMnFBLWQBukcDnrJ0VqjgoD_A/view?usp=sharing


use the link above.

thanks

**Comment by ayotade** [comment, marked as answer]
PetplanetAmazonApp.zip
<https://drive.google.com/file/d/1tLjclCcEGVrY21verC0cML6uwVJp1wmh/view?usp=drive_web>


please ignore the previous version. thank you
